examples/predicted_molecular_consequences/rs699.json

```diff
@@ -0,0 +1,260 @@
+{   
+    "name": "rs699",
+    "alleles":  [{
+        "name": "NC_000001.1:230710048:A:G",
+        "allele_type": 
+        {
+            "accession_id": "SO:0001483",
+            "value": "SNV",
+            "url": "www.sequenceontology.org/browser/current_release/term/SO:0001483",
+            "source": {
+            "id": "...",
+            "name": "Sequence Ontology",
+            "url": "www.sequenceontology.org",
+            "description": "The Sequence Ontology..."
+            }
+        },
+        "predicted_molecular_consequences": 
+        [{
+            "allele_name": "G",
+            "stable_id": "ENST00000680783",
+            "feature_type" : 
```

Comment:
json breaks here (same in other consequences)

Reply:
Hi @azangru , is there a field that is breaking? I am not seeing issues when I paste it on an online JSON viewer

Reply:
Oh, sorry, my fault. I expected the `feature_type` field to be of type string, which got me confused; I did not realise that the object starting on the next string was intended as its value.

Is there any reason for `feature_type` not to be string? You can see [in CDM](https://github.com/Ensembl/ensembl-cdm-docs/blob/main/src/docs/feature.md) that types of things are strings.

Reply:
> Is there any reason for `feature_type` not to be string? You can see [in CDM](https://github.com/Ensembl/ensembl-cdm-docs/blob/main/src/docs/feature.md) that types of things are strings.

From my knowledge, the reason it is `OntologyTermMetadata` is to capture the SO term. Current design view does not seem to require the SO term but I will have to check if that is the case for all types of features.